frontend/src/components/register/RegisterInput.js

```diff
@@ -110,6 +110,12 @@ const RegisterInput = () => {
 		}
 	};
 
+	const handleKeyPress = (e) => {
+		if (e.keyCode === 13) {
+			handleClick();
+		}
+	};
+
```

Comment:
Isn't there a way to avoid using this?

Reply:
Not sure, tried multiple approaches, none of them worked.